README.md

```diff
@@ -125,7 +137,13 @@ Write code to do the following:
 
 Write your code here:
 ```ruby
-# code here
+# assuming the lamp and genie templates are automatically assigning ids
+genie = Genie.create(name: "Genie")
+lamp = Lamp.create(wishes_remaining: 3, genie: genie.id)
```

Comment:
In this case, I think the relationship is flipped, so a `genie` belongs_to a `lamp`
therefore the `genie` would have an foreign_key corresponding to a `lamp`'s id.

Therefore we can just do: 
```ruby 
lamp = Lamp.create(wishes_remaing: 3)
lamp.genie.create(name: "Genie")
```